src/apps/intel/intel10g.lua

```diff
@@ -187,6 +192,21 @@ do
    end
 end
 
+function M_sf:poll_ingress_packet_drops ()
+   self.qprdc = 0
+   local function fn ()
+      self.qprdc = tonumber(self.qs.QPRDC[0]())
+   end
+   local interval = 1e9 * ingress_packet_drops.poll_timer
+   local t = timer.new("ingress_packet_drops", fn, interval, 'repeating')
+   timer.activate(t)
+   return self
+end
+
+function M_sf:ingress_packet_drops ()
+   return self.qprdc
```

Comment:
The reason why resetting the value to zero was that the value of QPRDC is stored in self.qprdc but it only gets updated every second. The engine polls the value several times per second so in the case it needs to jit.flush it will do it continuously until a new QPRDC value from the card is read, that's the reason to set it to zero once read.

It slipped my mind to do the same for SF configurations.

If we skip the timer and read QPRDC continuously resetting to zero after a read won't be necessary. I will measure perf of this PR and without the timer. Let's see how it goes.